### PR TITLE
Issue 1 10 fix

### PR DESCRIPTION
**Pull request by blue32captain**
fixed issues #1, #3, #5, #7

### CODE REVIEW

**Comment by oishun1112** [comment]
https://github.com/code-423n4/2022-01-insure-findings/issues/1
https://github.com/code-423n4/2022-01-insure-findings/issues/3
https://github.com/code-423n4/2022-01-insure-findings/issues/5
https://github.com/code-423n4/2022-01-insure-findings/issues/7

File: test/unitary/IndexTemplate/unitIndex.test.js [hunk 1]
```diff
@@ -2083,26 +2083,27 @@ describe("Index", function () {
...
 
-      let expectedWithdrawable = (await index.totalLiquidity()).sub(sum);
+      let expectedWithdrawable = indexLiquidity.sub(lockedAmount);
       let withdrawable = await index.withdrawable();
 
       expect(withdrawable).to.equal(expectedWithdrawable);
     });
 
-    it("should return zero when _leverage > targetLev + upperSlack", async function () {
+    it.skip("should return zero when _leverage > targetLev + upperSlack", async function () {
```
Comment (oishun1112) [inline test/unitary/IndexTemplate/unitIndex.test.js:2106]: please explain about this skip

**Comment by blue32captain** [comment]
withdrawable() function was changed when pushed from code4rena repo to the branch.
that test case is for original.

**Comment by oishun1112** [comment]
I see. please dismiss the test then. I'll come up with the new test case

**Review by oishun1112 — APPROVED** [review]
test passed

**Pull request merged**